scripts/repos.py

```diff
@@ -23,6 +23,7 @@
 import sys
 from typing import Dict, List, Any
 
+# pylint: disable=import-error
```

Comment:
This does not look right. Are you running pylint from a virtual environment with installed dependencies?

Reply:
Yes, I'm running in a virtual environment but it appears to be misconfigured. I'll remove this line.